lms/djangoapps/class_dashboard/tests/test_dashboard_data.py

```diff
@@ -316,5 +316,5 @@ def test_has_instructor_access_for_class(self):
         """
         Test for instructor access
         """
-        ret_val = has_instructor_access_for_class(self.instructor, self.course.id)
+        ret_val = bool(has_instructor_access_for_class(self.instructor, self.course.id))
```

Comment:
Rather, change the test to use `self.assertTrue` instead of `self.assertEquals`.  `assertTrue` and `assertFalse` are more specific assertions and provide better error messages if they fail.